python/spear/config/default_config.simulation_controller.yaml

```diff
@@ -20,11 +20,19 @@ SIMULATION_CONTROLLER:
   CUSTOM_UNREAL_CONSOLE_COMMANDS: []
 
   CAMERA_AGENT:
+    CAMERA_ACTOR_NAME: ""
     ACTION_COMPONENTS: ["set_pose", "set_num_random_points"] # "set_pose", "set_num_random_points"
     OBSERVATION_COMPONENTS: ["camera"] # "camera"
     STEP_INFO_COMPONENTS: ["random_points"] # "random_points"
+    SPAWN_MODE: "specify_pose" # "specify_pose", "specify_existing_actor"
```

Comment:
Would it be better to keep the default as empty string? This way, user has to define a certain spawn mode, and combining this with other consequent parameters like spawn position, orientation, etc is more clear in config yamls. In our `user_config.yaml.example` file, we can keep the default as `"specify_poses"`.

Reply:
I like `specify_pose` as the default, because I think spawning an actor at `[0,0,0]` is a reasonable default (this is the existing behavior), and that is what you'll get if you don't specify anything extra in `user_config.yaml`. Also, actors will sometimes be re-positioned by the task anyway, so I wouldn't want to force those users to specify more stuff than necessary in their `user_config.yaml`.